mhs/outbound/outbound/request/synchronous/request_body_schema.py

```diff
@@ -0,0 +1,53 @@
+import dataclasses
+from typing import List
+
+import marshmallow.validate
+
+# These allowed content types for attachments are taken from the EIS part 2.5.4.2
+_ATTACHMENT_ALLOWED_CONTENT_TYPES = (
+    'text/plain', 'text/html', 'application/pdf', 'text/xml', 'application/xml', 'text/rtf', 'audio/basic',
+    'audio/mpeg', 'image/png', 'image/gif', 'image/jpeg', 'image/tiff', 'video/mpeg', 'application/msword'
+)
```

Comment:
@colin-bateson In the EIS part 2.5.4.2 there are 2 tables, one has `application/octet-stream` and the other doesn't. Do we know whether this is a content-type that we should allow?

Reply:
I'd say include it rather than exclude it